src/utils/json/messages.json

```diff
@@ -1930,5 +1930,150 @@
 		"pt-BR": ":ledger: Ministro",
 		"en-US": ":ledger: Minister",
 		"es-ES": ":ledger: Ministro"
+	},
+	"FARM_TITLE": {
+		"pt-BR": "🏞️ Fazenda de {USER}",
+		"en-US": "🏞️ {USER}'s Farm",
+		"es-ES": "🏞️ Granja de {USER}"
+	},
+	"WATERING_TIP": {
+		"pt-BR": "💦 **Regue as platações com `/fazenda regar` para diminir seu tempo de crescimento!**",
+		"en-US": "💦 **Water your crops with `/farm water` to decrease their growth time!**",
+		"es-ES": "💦 **¡Riega tus cultivos con `/granja regar` para disminuir su tiempo de crecimiento!**"
```

Comment:
This is a good message, I would just remove the `/farm water`, since there is a button to water just below, and it makes the embed horizontally longer